airflow/cli/simple_table.py

```diff
@@ -53,13 +56,18 @@ def print_as_table(self, data: List[Dict]):
             table.add_row(*[str(d) for d in row.values()])
         self.print(table)
 
-    def _normalize_data(self, value: Any, output: str) -> Union[list, str, dict]:
+    # pylint: disable=too-many-return-statements
+    def _normalize_data(self, value: Any, output: str) -> Optional[Union[list, str, dict]]:
         if isinstance(value, (tuple, list)):
             if output == "table":
                 return ",".join(self._normalize_data(x, output) for x in value)
             return [self._normalize_data(x, output) for x in value]
         if isinstance(value, dict) and output != "table":
             return {k: self._normalize_data(v, output) for k, v in value.items()}
+        if isinstance(value, PluginsDirectorySource):
+            return str(value)
+        if inspect.isclass(value):
+            return value.__name__
```

Comment:
I understand that we have these two `IF`s in order here because `PluginsDirectorySource ` is a special `class` to handle (we want to use its `__str__` rather than `__name__`). But the first glance at it is a bit confusing because what it returns _looks the same_ to the last return line. This may be a confusion to people who read this later.

So I would like to suggest changing it to

```python
if inspect.isclass(value) and not isinstance(value, PluginsDirectorySource):
    return value.__name__
```